src/main/java/io/cdap/e2e/pages/actions/CdfPipelineRunAction.java

```diff
@@ -109,7 +119,16 @@ public static void waitForPipelineToTransitionToStatus(String pipelineStatus, lo
    * Wait till the Pipeline's status changes (from Running) to either Succeeded, Failed or Stopped within the
    * Timeout: {@link ConstantsUtil#PIPELINE_RUN_TIMEOUT_SECONDS}
    */
-  public static void waitTillPipelineRunCompletes() {
+  public static void waitTillPipelineRunCompletes() throws InterruptedException, IOException {
+    // Adding a page refresh in case tests are running on CDF to update the pipeline status.
+    if (Boolean.parseBoolean(SeleniumHelper.readParameters(ConstantsUtil.TESTONCDF)) ||
+      Boolean.parseBoolean(SeleniumHelper.readParameters(ConstantsUtil.TESTONHDF))) {
+      do {
+        PageHelper.refreshCurrentPage();
+        SeleniumDriver.getWaitDriver(ConstantsUtil.PAGE_LOAD_TIMEOUT_SECONDS);
```

Comment:
A page refresh in every `50seconds` might be too frequent, it won't change the pipeline status drastically, I would suggest adding a sleep of atleast `2 mins`.

Reply:
Updated the sleep time to 2mins, re-using ```PIPELINE_DEPLOY_TIMEOUT_SECONDS``` as it's value is 120, but the constant name can create some confusion here.

Should we create a new constant with same value named as ```PIPELINE_REFRESH_TIMEOUT_SECONDS```  

(https://github.com/cdapio/cdap-e2e-tests/pull/197/commits/44be5d5c5c9bf2c3ee877d3eb7dd3aa43c414929#diff-cfa45894e4592a5d6ed32f485ed2594ae01b9a3b154e60820a599164e0b52e80R136)

Made some changes to this function PTAL